README.md

```diff
@@ -6,11 +6,22 @@ The goal is to study the features required for such identification, when working
 the code of this project is arrange according to the architecture shown in https://github.com/asilev/NLP_FinalProject/blob/master/doc/architecture/NLP%20Author%20Classification%20Architecture.png.
 
 The files in TwitterReader are used to retrieve data sets from Twitter, using the twitter4j framework.
-This folder is meant to be compiled and run as a standalone executable with its own main method.
+The Twitter4j framework provides a pre-compiled JAR library, twitter4j-core-4.0.4.jar, which is imported into the project and then allows the access and extraction of data.
 Running it requires two parameters: The inpus path (this is unused, and can be anything) and the output path (where to put the tweets from tweeter).
```

Comment:
I'm not sure I'm getting it right, but I think the first parameter is useless. If so, it is better to remove it from code, and change the instruction to explain only the remaining parameter.